### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Remote Debugging-Debugger_Keyboard: Unable to access "Choose the language dropdown" and "set value dropdown" using keyboard 

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3600]
**GitHub Tags:** #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.1.1; 

[**Check out Accessibility Insights!**](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 2.1.1

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala)
3)Tab till next button and press enter
4)Fill all the details correctly and Tab till finish button and press enter
5)Tab till -->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
6)Tab till Edit Configurations menu(select the icon in the upper-right corner)
7)Navigate to Advanced Configuration (Remote Debugging) section and Press the shortcut key to check Enable Spark remote debug
8)Fill Serure Shell (SSH) User Name and SSH password
9)Sent some breakpoints in SparkCore_wasbloTest code and tab till sparkjobdebug and press enter
10)Press cntrl+tab to navigate to Debug Remote Spark Job in Cluster section 
11)Naviagte to variables tab and open the context menu for any of the variables by using keyboard
12)Tab till "set value" in the context menu and press enter. Verify user is able to access "Choose the language dropdown" and "set value dropdown" using keyboard or not

**Actual Result:**
User is not able to access "Choose the language dropdown" and "set value dropdown" using keyboard 

**Expected Result:**
User should be able to access "Choose the language dropdown" and "set value dropdown" using keyboard 

**User Impact:**
Keyboard users will find difficult to change the language and to see the dropdown items if they are not accessible using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:
![Bug_3600_unable to access dropdown](https://user-images.githubusercontent.com/42638450/65422770-02953580-de25-11e9-9865-bca0ecf67c59.png)
**

**Comment by t-rufang** [comment]
This feature is supported by IntelliJ.

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749703
